### ISSUE TITLE
Using the plugin with WordPress installed in a sub-directory

### ISSUE THREAD

**Issue by LorisOclock** [usefulteam/jwt-auth#2]
Hi @contactjavas 

With my team, we found out where the problem (https://wordpress.org/support/topic/whitelisting-token-generation-endpoint/) came from. It is due to our custom installation with WordPress in a sub-directory (https://wordpress.org/support/article/giving-wordpress-its-own-directory/).

To make it work in this case too,

https://github.com/usefulteam/jwt-auth/blob/91b2a59fd106b4a444e15b8f8dcd4100cfbc1d7e/class-auth.php#L453
should be
```php
if ( home_url('/' . $this->rest_api_slug,'relative') . '/jwt-auth/v1/token' !== $request_uri ) {
```

**Comment by contactjavas** [comment]
Wow! thanks @LorisOclock  :)
Great finding. Thank you for debugging it and posting the solution.
I'll implement it :)

**Comment by contactjavas** [comment]
Thanks for your time debugging & reporting this @LorisOclock .
It's implemented in [v1.4.0](https://wordpress.org/plugins/jwt-auth/)
